human-readable-types/src/main/java/com/palantir/humanreadabletypes/HumanReadableByteCount.java

```diff
@@ -245,6 +246,7 @@ public int hashCode() {
      * @return a human-readable string representation of this byte string, not null
      */
     @Override
+    @JsonValue
```

Comment:
Matches `HumanReadableDuration` which already applies `@JsonValue` to `toString()` :+1: